.env

```diff
@@ -0,0 +1,2 @@
+CTF_SPACE_ID=0123456789ab
+CTF_CDA_ACCESS_TOKEN=0123456789abcdefghijklmnopqrstuvwxyz0123456789abcdefghijklmnopqr
```

Comment:
These tokens are simply placeholder. Should the `.env` file be ignored by default, allowing local environments to define their own values? I just want to avoid a situation where I accidentally commit keys to the repo because I edited this file then carelessly ran `git add .` or something similar.

Reply:
The `.env` file is read by `docker-compose` when you run it.
You can therefore overwrite the env vars already defined in the image.
Each environment will have in the stack repo a separate `.env` file.